string class_id
float32 class_support
string object_id
float32 object_support
string name
string orientation
